aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -505,6 +505,60 @@ void minimum_kernel(TensorIterator& iter) {
   }
 }
 
+void fmax_kernel(TensorIterator& iter) {
+  if (iter.dtype() == ScalarType::Bool) {
```

Comment:
There are a few updates needed here:

- use iter.common_dtype() to test for whether it's a floating type or not and to dispatch
- if iter.common_dtype() is not a floating point type then just call maximum 
- in the floating point kernel I don't think you need the NaN checks, I think this can just call std::fmax

Similar changes will need to be made for fmin.

Reply:
Gotcha, I'll fix it right away.

Reply:
Addressed!